packages/compass/package.json

```diff
@@ -129,7 +129,7 @@
   },
   "scripts": {
     "install": "node scripts/download-fonts.js && node scripts/download-csfle.js",
-    "electron-rebuild": "electron-rebuild --only kerberos,keytar,interruptor,os-dns-native,win-export-certificate-and-key,macos-export-certificate-and-key --force --prebuild-tag-prefix not-real-prefix-to-force-rebuild",
```

Comment:
We don't need force here (iirc historically this command was used during the actual packaging process, but it's not a thing anymore), it just slows down restart for no good reason